webapi/src/test/scala/org/knora/webapi/responders/v2/ValuesResponderV2Spec.scala

```diff
@@ -4244,7 +4254,7 @@ class ValuesResponderV2Spec extends CoreSpec() with ImplicitSender {
       val valueContent = StillImageFileValueContentV2(
         ontologySchema = ApiV2Complex,
         fileValue = FileValueV2(
-          internalFilename = MockSipiConnector.FAILURE_FILENAME, // tells the mock Sipi responder to simulate failure
+          internalFilename = "failura.jp2", // tells the mock Sipi responder to simulate failure
```

Comment:
this looks strange to me

Reply:
also, this should still use the constant rather than the string literal.

And there is a typo in this string - so the fact that this test passes is a bis unsettling to me.